internal/cache/file/cache_handle_test.go

```diff
@@ -820,3 +820,31 @@ func (cht *cacheHandleTest) Test_MultipleReads_CacheHitShouldBeFalseThenTrue() {
 	assert.True(cht.T(), cacheHit)
 	assert.Nil(cht.T(), err)
 }
+
+func (cht *cacheHandleTest) Test_Read_Sequential_Parallel_Download_True() {
+	dst := make([]byte, ReadContentSize)
+	offset := int64(cht.object.Size - ReadContentSize)
+	cht.cacheHandle.isSequential = true
+	cht.cacheHandle.cacheFileForRangeRead = true
+
+	fileDownloadJob := downloader.NewJob(
+		cht.object,
+		cht.bucket,
+		cht.cache,
+		DefaultSequentialReadSizeMb,
+		cht.fileSpec,
+		func() {},
+		&config.FileCacheConfig{EnableCrcCheck: true, EnableParallelDownloads: true},
```

Comment:
pass some values to `DownloadParallelismPerFile` and `ReadRequestSizeMb`

Reply:
done